pkg/security/scc/byrestrictions.go

```diff
@@ -75,10 +78,40 @@ func volumePointValue(scc *kapi.SecurityContextConstraints) int {
 	}
 
 	if hasHostVolume {
-		return 10
+		return 10000
 	}
 	if hasNonTrivialVolume {
-		return 5
+		return 5000
 	}
 	return 0
 }
+
+// hasCap checks for needle in haystack.
+func hasCap(needle kapi.Capability, haystack []kapi.Capability) bool {
+	for _, c := range haystack {
+		if needle == c {
+			return true
+		}
+	}
+	return false
+}
+
+// capabilitiesPointValue returns a score based on the capabilities allowed,
+// added, or removed by the SCC.
+func capabilitiesPointValue(scc *kapi.SecurityContextConstraints) int {
+	points := 500
+	points += 30 * len(scc.DefaultAddCapabilities)
+	if hasCap(kapi.CapabilityAll, scc.AllowedCapabilities) {
+		points += 300
+	} else {
+		points += 10 * len(scc.AllowedCapabilities)
+	}
+	points -= 50 * len(scc.RequiredDropCapabilities)
+	if (points > 1000) {
+		return 1000
```

Comment:
The goal was not to interfere with the other logic ... but that caps us at 10000, not 1000, will fix.